southpark-search/tests/distributed/terraform.tf

```diff
@@ -107,6 +91,16 @@ data "aws_iam_policy_document" "assume_role_policy" {
   }
 }
 
+resource "aws_cloudwatch_log_group" "southpark_loggroup" {
+  name              = "southpark-log"
+  retention_in_days = 1
+}
+
+resource "aws_cloudwatch_log_stream" "southpark-distributed-logs" {
+  name           = "southpark-distributed-logs"
```

Comment:
Do we plan on using this log stream of distributed logs?

Reply:
Yeah I think it would make sense or?